apps/openmw/mwgui/mainmenu.cpp

```diff
@@ -285,15 +285,18 @@ namespace MWGui
             Gui::ImageButton* button = mButtons[buttonId];
             button->setVisible(true);
 
+            // By default, assume that all menu buttons textures should have 64 height.
+            // If they have a different resolution, scale them.
             MyGUI::IntSize requested = button->getRequestedSize();
+            float scale = requested.height / 64.f;
 
-            button->setImageCoord(MyGUI::IntCoord(0, 0, requested.width, requested.height));
+            button->setImageCoord(MyGUI::IntCoord(0, 8*scale, requested.width, requested.height - 8*scale));
```

Comment:
no what are you doing stop

Reply:
Hardcoded values?

Reply:
No, it reintroduces the offset that cuts off actually necessary parts of the button texture.

Reply:
We already have the cutoff here.

Reply:
Anyway, I removed the cutoff now.